packages/sampler/stories/qa/Spinner.js

```diff
@@ -15,6 +17,38 @@ const
 		longText:'SpinnerWithLongText SpinnerWithLongText SpinnerWithLongText'
 	};
 
+class FocusOnSpinner extends React.Component {
+	constructor (props) {
+		super(props);
+		this.state = {
+			txt: '',
+			isLoading: false
+		};
+	}
+
+		handleChange = (e) => {
+			this.setState({
+				txt: e.value
+			});
+		}
+
+		handleDeactivate = () => {
+			this.setState({
+				isLoading: true
+			});
+		}
+
+		render () {
+			return (
+				<div>
+					<Button>open</Button>
```

Comment:
Does this `Button ` serve a purpose?  Can you add some text to describe how this test is supposed to work?

Reply:
Removed